### PR TITLE
Makes gearing up as sec related jobs less tedious

### PR DESCRIPTION
**Pull request by Sarcalogo**
So this PR does a lot of minor things that make gearing up as Sec a lot
easier:

- Security belts spawn loaded with handcuffs, stun baton, flashbang, pepperspray, and a sec light.

- All security now start with a flash and handcuffs in their left and right pockets. Security Officers lose the stun baton in their backpack, mostly to prevent stun batons from being everywhere.

- Removes flashes, stun batons, pepper spray, and sec light from sec lockers.

- Removes telescopic baton from the HoS backpack and places it in the locker if he wants to use it. (most don't) 

- Removes the stun baton from HoS locker (it's in the security belt)

- Adds three black gloves to the Security wardrobe, there were none before.

- Changes the order of the items in lockers to make the first item in the locker different. Makes clicking and equipping as fast as possible:

- Sec lockers are now in order: HUDS, security belt, helmet, junk.

- Captain's locker is now in order: HUD's, gloves, energy gun, junk.

- The HoS locker is now in order: HoS gun, belt, telescopic items, loyalty implants, junk.

- HoP's locker is now in order: Sunglasses, armor, gun, flash, junk.

I think that's about it..

### CODE REVIEW

**Comment by Gandalf2k15** [comment]
wot

**Comment by Sarcalogo** [comment]
Refresh the page Gandalf old buddy.

**Comment by Sarcalogo** [comment]
An example.. I kind of misclick the sec belt but whatever.

https://my.mixtape.moe/vukkiy.webm

**Comment by McBawbaggings** [comment]
I like it, makes gearing up less of a chore and it also makes sec more of a juicy target with fully stocked belts. Also makes people who catch late join sec not have ez access to stun batons until they ransack security itself.

**Comment by Gandalf2k15** [comment]
+6999999

**Comment by Kokojos** [comment]
Yes

**Comment by Sarcalogo** [comment]
The HoS locker is probably the hardest one to make simpler. 

I don't know how many take the armored great coat for example, or how many people use the telescopic shield, or how many use the swat mask. I tried to focus on the weapons primarily for that.

**Comment by calibraptor** [comment]
Yeah sure +1

As for the HoS locker why fix what isn't broken?

**Comment by Nebyab** [comment]
I'm a little iffy on having items in sec pre-loaded by default. With that said this is one of those things I think a trial of would be for the best.

**Comment by dreamcrab** [comment]
i don't really think this is necessary, personally

**Comment by Sarcalogo** [comment]
Reducing tedium is not unnecessary. It rearranges the order of items in the locker so you can sequentially pick up the most important items without alt clicking or right clicking. The other change is the automatic filling of Security belts. The only place to get a security belt is in a security locker, which previously held all the items that were present in the locker. All it does is make it so you don't have to manually load the belt, saving you some time. There is absolutely no reason at all any of these changes would cause problems, only good things, reducing tedious gearing up.

**Comment by Spacedong** [comment]
nah I think this is pretty OP, gearing up in security is a tribal ritual since olden times, don't do this, just get good

**Comment by Sarcalogo** [comment]
"The tedium is a feature and tradition, so we can't abandon it." - Spacedong

**Comment by Carbonhell** [comment]
Don't remove the telebaton from HoS backpack, think about latejoin hoses too. All heads come with a baton, so i think the HoS should have it aswell,i personally use it.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -107,19 +105,15 @@
...
-	new /obj/item/weapon/grenade/flashbang(src)
+	new /obj/item/device/radio/headset/headset_sec/alt(src) 
+	new /obj/item/clothing/head/helmet/sec(src)
 	new /obj/item/weapon/storage/belt/security/full(src)
 	new /obj/item/clothing/glasses/hud/security/sunglasses(src)
-	new /obj/item/clothing/head/helmet/sec(src)
 
 /obj/structure/closet/secure_closet/security/sec
 
 /obj/structure/closet/secure_closet/security/sec/New()
```
Comment (Carbonhell) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm:116]: What is this object used to then?It has no difference whatsoever with its parent

**Comment by Sarcalogo** [comment]
I have no idea, I thought it was odd too how it just stood out there with one item inside it. No clue. Gonna have to ask /tg/ what they were thinking.

**Comment by Sarcalogo** [comment]
Yeah but the HoS starts with an energy gun, flash, and cufs, when he late joins. Doesn't really need the baton. I suppose I could change it so he starts with a stun baton, but I think it's dumb.

**Comment by Carbonhell** [comment]
I forgot about the energy gun, my bad. Baton isn't needed then, just remove that /sec children and its proc and this pr is good to go after another head okays it

**Comment by Sarcalogo** [comment]
Not sure what you mean when you say sec children and it's proc. You mean this right?

/obj/structure/closet/secure_closet/security/sec
 
 /obj/structure/closet/secure_closet/security/sec/New()

**Comment by Sarcalogo** [comment]
Actually CarbonHell, looking at the code I think this is responsible for spawning in new lockers in Security? By removing that line of code a compile error happened so... what the heck!!

**Comment by Sarcalogo** [comment]
Anyway Carbon, it looks like the items spawned on the map are of the security/sec type instead of just plain security type. I will be changing this back to avoid compile errors and mapping issues.

**Comment by Carbonhell** [comment]
Shrug alright

**Comment by Voratius** [comment]
sure, equiping is needlessly tedious.

+1/3rd

**Comment by Kokojos** [comment]
currently standing at 3gm and 1 HA voting for, 2 other HA votes are unclear. Teamstab and neb if you wanna disagree cmon here.

**Comment by blobbernaut** [comment]
Agree with all changes except

 Removes flashes, stun batons, pepper spray, and sec light from sec lockers.
The lockers should have their ID locks removed in that case, since there's no real usable loot in them

Because a security officer has their stun baton in their backpack removed and added to the belt, this just results in them having less batons. Should keep the baton in there and remove the other things so that there's still incentive to get one of those lockers open.

**Comment by Sarcalogo** [comment]
Uh.. the lockers start with a security belt that has all of those things, they were not REMOVED but moved onto the belt.

**Comment by blobbernaut** [comment]
Uh.. my misunderstanding then, agree with everything here.

**Comment by Nebyab** [comment]
You know, thinking about it. Spacedong is kinda right imo (for the first time). Gearing up is part of the job, standardizing it would make it all lose something I fear.

**Comment by blobbernaut** [comment]
Maybe so, nebyab, but I still think everything past the first three bullet points are good changes even if you think that.

**Comment by Nebyab** [comment]
That's why I didn't say -1 mate.

I voiced my concerns with an aspect of the pr.

**Comment by Kokojos** [comment]
This stands at 3 gm votes and 1 HA vote for, 1 HA againts, one unclear.
Il merge this within the day unless im stopped.

**Comment by GeneralGoose** [comment]
shieet 
but what about plebs robbing sec lockers? that shit is a must dude

**Comment by Sarcalogo** [comment]
Half of the lockers in sec aren't used, and most lockers outside of sec aren't looted by security. You're stil able to loot them. Anyway, this conflicts with Calibraptor's tweak, rather his conflicts with mine. It's probably ready to be merged too!

**Comment by Kokojos** [comment]
Well fuck, blame me if you want, I went top to bottom and merged his first.

**Comment by JohnGinnane** [comment]
Code looks good

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm (hunk 1 above)
Comment (JohnGinnane) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm:116]: Checked the map and `/obj/structure/closet/secure_closet/security/sec` and definitely used in a few places. I think having the New() override is okay incase people want to add in items in the future

**Comment by Sarcalogo** [comment]
It says it has conflicts but it doesn't, I just mirrored what Calibraptor did in my PR.

**Comment by Gandalf2k15** [comment]
You have to update from Hippie master and merge the changes. Its the only way it can be sorted. I do hope you use git client.

**Comment by Sarcalogo** [comment]
No I don't. 

You can look at the code and verify that nothing "really" conflicts even though it says it does.. I'm not doing all that bullshit, especially when early on I pointed out that his PR conflicted with mine. I  said this this on both PR's. If anyone is expected to fix their shit, it should be Calibraptor not me, but hey I did it anyway. 

You can merge PR's that "conflict" /tg/ does it all the time.

**Comment by calibraptor** [comment]
Wat I do

**Comment by Gandalf2k15** [comment]
Github thinks there is something wrong. Try editing the code slightly.

**Comment by Kierany9** [comment]
Last time this happened to me I just made a new PR, won't take too long since it's only 5 files.

**Comment by Sarcalogo** [comment]
Nothing you did, technically speaking falls on Kokojo. You made a PR like 2 weeks after I did, that involved the sec lockers and sec loadout. The same day it was posted I said it conflicted with my sec change PR (this one). So Kokojo merged yours first, despite both being approved, and now I'm stuck with the conflicts. I even remember joking in OOC that, that would happen.

I already fixed it, was very easy. Now Github says it conflicts (it doesn't) and Gandalf is saying I have to change a bunch of stuff (I don't, you can merge PR's with conflicts as evidenced by your PR being pulled while mine was still open). Just more beauracratic nonsense.

**Comment by Sarcalogo** [comment]
I no longer have an interest in contributing.

**Pull request closed without merging**